extensions/2.0/Vendor/EXT_feature_metadata/schema/featureIdTexture.schema.json

```diff
@@ -2,15 +2,19 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Feature ID Texture",
   "type": "object",
+  "allOf": [ { "$ref": "textureInfo.schema.json" } ],
   "description": "An object describing a texture used for storing per-texel feature IDs.",
   "properties": {
+    "index": { },
+    "texCoord": { },
```

Comment:
What's the purpose of these lines beyond the `allOf` line? just wondering.

Reply:
Good question – I'm not sure, but they occur in all other cases where `textureInfo` is extended so I imitated those:

- https://github.com/KhronosGroup/glTF/blob/c6a5521ebd1469aba076949a97f150bf3591b63f/specification/2.0/schema/material.occlusionTextureInfo.schema.json#L7-L8
- https://github.com/KhronosGroup/glTF/blob/c6a5521ebd1469aba076949a97f150bf3591b63f/specification/2.0/schema/material.normalTextureInfo.schema.json#L7-L8